src/App.js

```diff
@@ -163,6 +163,7 @@ function App() {
           path="/"
           element={
             <div className="todo-wrapper">
+              <img src="images/mountains.jpg" alt="" />
```

Comment:
The `alt` attribute is used to provide an alternative text for an image. Is meant to inform the content or purpose of the image in case the image cannot be displayed. Main reasons to use it are accessibility, SEO and user experience.